css/main.scss

```diff
@@ -77,5 +77,6 @@
 @import 'unsupported-browser/main';
 @import 'modals/invite/add-people';
 @import 'deep-linking/main';
+@import 'navigate_section_list';
```

Comment:
So the recent list scss is not used at all?

Reply:
Its not it should have been removed sorry about that.